src/server.c

```diff
@@ -5414,6 +5424,7 @@ void createPidFile(void) {
 }
 
 void daemonize(void) {
+    int log_to_stdout = server.logfile[0] == '\0';
     int fd;
```

Comment:
swap these 2 for the ancient-C coding style?..

Reply:
I don't think this breaks any current convention, though? Originally C did not allow mixing declarations (for function scope variables that might mean at most that stack space is reserved, but not initialized), in between statements. But initializing those variables has always been possible.

As an aside, C99 and some compiler extensions (most prominently GNU) has allowed that mixing for a long time though. But then, even now with prevalent compiler support, this is just a coding style. :)